spec/graphql/language/generation_spec.rb

```diff
@@ -7,11 +7,13 @@
       GraphQL.parse('type Query { a: String! }')
     }
 
-    class CustomPrinter < GraphQL::Language::Printer
```

Comment:
Note: This class seems defined at the top-level. In other words, the `CustomPrinter` in `spec/graphql/language/nodes_spec.rb` is identical to this class. I think it is not intended result because this class is defined under the `describe` block. For this reason I changed from the ordinary class definition to `Class.new` definition inside the `let`.

Reply:
> not intended 

I agree! Sometimes it's hard to forget that `describe ...` blocks are top-level. Thanks for fixing it!